compact/tracker/mpgd_barrel.xml

```diff
@@ -49,6 +49,12 @@
     <constant name="MMDriftCuElectrode_thickness"           value="5*um"/>
     <constant name="MMDriftKapton_thickness"                value="250*um"/>
     <constant name="MMDriftCuGround_thickness"              value="0.41*um"/>
+    
+    <comment> FIXME: No support material is here, so fudge factor used to bring material budget to ~0.5% for barrel </comment>
+    <constant name="MMFudgeInnerMPGDBarrel_thickness"                      value="570*um"/>
+
+    <comment> FIXME: No support material is here, so fudge factor used to bring material budget to ~1% for barrel </comment>
+    <constant name="MMFudgeOuterMPGDBarrel_thickness"                      value="2.50*mm"/>
```

Comment:
Can you file a new issue corresponding with the FIXMEs here?

Reply:
I will file this issue soon, along with a couple others